src/hist.jl

```diff
@@ -97,7 +98,7 @@ Plots.group_as_matrix(g::GroupedHist) = true
             v_i = filter(x->!isnan(x), v[:,i])
             w_i = weights == nothing ? nothing : weights[groupinds]
             h_i = Plots._make_hist((v_i,), h.edges; normed = false, weights = w_i)
-            y[:,i] = normed ? h_i.weights / sum(h.weights) : h_i.weights
+            y[:,i] = normed ? (h_i.weights / sum(h_i.weights)) .* (length(v_i) / ntot) : h_i.weights
```

Comment:
This eliminates two allocations:
```suggestion
            if normed
                y[:,i] .= h_i.weights .* (length(v_i) / ntot / sum(h_i.weights))
            else
                y[:,i] .= h_i.weights
            end
```